docs/python/inference/api_summary.rst

```diff
@@ -67,8 +104,6 @@ use IOBinding to put input on CUDA as the follows.
 	session.run_with_iobinding(io_binding)
 	Y = io_binding.copy_outputs_to_cpu()[0]
 
-Scenario 2:
```

Comment:
I would prefer to mention these sections as scenarios because that's what they really represent - different use cases and how IOBinding can be used in those cases. I found the deletion of Scenario a bit confusing. May be we can call it a different name like Use Case?

Reply:
I don't have a problem with the word scenario - it was more that I felt like the numbering didn't add anything. If we are breaking  this functionality into a number of canonical scenarios then it would be helpful to name them.

Reply:
Sure, I guess it's easier to number them and describe the scenario than coming up with good headings/titles. The main idea is to show different scenarios and how to use IO binding in those scenarios.